.github/workflows/scripts/issues/titleCleanup.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+const regexes = {
+  // covers 'dataGrid' in all variants, as well as with suffixes (-pro/-premium/etc)
+  dataGrid: /\s?\[\s?(data\s?grid[-\sA-Za-z]*)]/gi,
+  // covers 'charts' in all variants, as well as with prefixes (PieChart, LineChart, etc)
+  charts: /\s?\[\s?([-\sA-Za-z]*charts?)]/gi,
+  // covers 'pickers' in all variants, as well as with prefixes (DatePicker, time-picker, etc)
+  pickers: /\s?\[\s?([-\sA-Za-z]*pickers?)]/gi,
+  // covers 'treeView' in all variants
+  treeView: /\s?\[\s?([-\sA-Za-z]*tree\s?(view|item))]/gi,
```

Comment:
```suggestion
  // covers 'dataGrid' in all variants, as well as with suffixes (-pro/-premium/etc)
  dataGrid: /\s*\[\s*(data\s?grid[-\sA-Za-z]*)]/gi,
  // covers 'charts' in all variants, as well as with prefixes (PieChart, LineChart, etc)
  charts: /\s*\[\s*([-\sA-Za-z]*charts?)]/gi,
  // covers 'pickers' in all variants, as well as with prefixes (DatePicker, time-picker, etc)
  pickers: /\s*\[\s*([-\sA-Za-z]*pickers?)]/gi,
  // covers 'treeView' in all variants
  treeView: /\s*\[\s*([-\sA-Za-z]*tree\s?(view|item))]/gi,
```

probably want to deal with an unknown amount of empty spaces if you care about that

Reply:
I normally try to prevent greedy, but in this case it might make sense.